### DISCUSSION TITLE
[Question]: Is there any way to share an agent with only a group of people?

### DISCUSSION THREAD

**Discussion by AhmedHalat** [danny-avila/LibreChat#4946] [Q&A] [answered]
As the title states, I'm looking for a way to create an agent but only share it with a specific group of people. One of the tools the agent has access to allows it to pull and format data that I don't want everyone using the app to have access to.

I saw a mention of workspaces in one of Danny's replies: [link to the comment](https://github.com/danny-avila/LibreChat/issues/3607#issuecomment-2447283147). However, I can't seem to find any more information on the subject. Can anyone provide more details or guidance on how to achieve this?

**Comment by danny-avila** [comment, marked as answer]
When this issue is implemented, it will be possible (planned):  https://github.com/danny-avila/LibreChat/issues/4730
